aviatrix/resource_aviatrix_transit_gateway.go

```diff
@@ -753,6 +753,31 @@ func resourceAviatrixTransitGateway() *schema.Resource {
 					},
 				},
 			},
+			"interface_mapping": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "List of interface names mapped to interface types and indices.",
```

Comment:
Is this sufficient documentation? I feel like I would need a couple of paragraphs of explanation to even understand whether I need to worry about this parameter at all.

Are there public documents that we could link to here?

Reply:
interface_mapping is an optional field. Users don't have to provide this value. The mapping value is expected only for esxi devices which is used in internal lab testing.